log.go

```diff
@@ -116,24 +301,11 @@ type activeEventKeyType struct{}
 
 var activeEventKey = activeEventKeyType{}
 
-// Stores an event in a context to be finished at a later point
-func (el *eventLogger) EventBeginInContext(ctx context.Context, event string, metadata ...Loggable) context.Context {
-	eip := el.EventBegin(ctx, event, metadata...)
-	return context.WithValue(ctx, activeEventKey, eip)
-}
-
-// Will complete an event if there is one in ctx
-func MaybeFinishEvent(ctx context.Context) {
-	val := ctx.Value(activeEventKey)
-	if eip, ok := val.(*EventInProgress); ok {
-		eip.Done()
-	}
-}
-
+// DEPRECATED use `Start`
 func (el *eventLogger) Event(ctx context.Context, event string, metadata ...Loggable) {
```

Comment:
Not sure how to replace logic here with the new methods. This is basically like calling `SetTag` or `LogKV`.

Reply:
Places like https://github.com/ipfs/go-ipfs/blob/master/exchange/bitswap/workers.go#L62